base.ini

```diff
@@ -53,13 +53,13 @@ set queue_worker_processes = 16
 set queue_worker_chunk_size = 1024
 set queue_worker_batch_size = 2000000
 
-[composite:regionindexer]
-use = egg:snovault#indexer
-app = app
-path = /index_file
-timeout = 60
-set embed_cache.capacity = 5000
-set regionindexer = true
+#[composite:regionindexer]
```

Comment:
I think really delete it if we're getting rid of it.

Reply:
ok, will delete in encoded as well